### PR TITLE
snort: update to 2.9.9.0 / libdaq: update to 2.0.6

### PR DESCRIPTION
**Pull request by mkrkn**
Maintainer: @lperkov 
Compile tested: (mpc85xx, tl-wdr4900, LEDE r4467)
Run tested: (mpc85xx, tl-wdr4900, LEDE r4467, starts up, loads engine, rules etc., ready to capture packets)

Description:
Update Snort and DAQ library to current versions. Does require tweaking snort.conf to device limits to avoid running out of memory.

### CODE REVIEW

**Comment by champtar** [comment]
Travis tests

File: libs/libdaq/Makefile [hunk 1]
```diff
@@ -8,13 +8,13 @@
...
-PKG_VERSION:=2.0.4
-PKG_RELEASE:=2
+PKG_VERSION:=2.0.6
+PKG_RELEASE:=1
 
 PKG_SOURCE_URL:=https://www.snort.org/downloads/snort/ \
 	@SF/snort
 PKG_SOURCE:=daq-$(PKG_VERSION).tar.gz
-PKG_MD5SUM:=65e51d72e9d5d8b397e192e4e5857eff
+PKG_MD5SUM:=b40e1d1273e08aaeaa86e69d4f28d535b7e53bdb3898adf539266b63137be7cb
```
Comment (champtar) [inline libs/libdaq/Makefile]: Please use PKG_HASH

File: net/snort/Makefile [hunk 2]
```diff
@@ -8,16 +8,16 @@
...
+PKG_RELEASE:=1
 
 PKG_LICENSE:=GPL-2.0
 PKG_MAINTAINER:=Luka Perkov <luka@openwrt.org>
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://www.snort.org/downloads/snort/ \
 	@SF/$(PKG_NAME)
-PKG_MD5SUM:=b01e9964827394c39194a0147bc15705
+PKG_MD5SUM:=fd3012bf36710481d66b40ad046b231d
```
Comment (champtar) [inline net/snort/Makefile]: Please use PKG_HASH (with a sha256)

**Comment by champtar** [comment]
Hi @mkrkn 
I've pushed a conflicting commit, can you rebase on top of latest master?
@lperkov please ack/nack

**Comment by mkrkn** [comment]
Hi @champtar 

I have now rebased on da11ac80ec8f6081b4708d19ef26c5733bbb6228.

**Comment by champtar** [comment]
Thanks @mkrkn, now waiting to hear from @lperkov

**Comment by diizzyy** [comment]
@champtar 
Commit it due to maintainer timeout.

**Comment by MikePetullo** [comment]
I am willing to take over maintaining the snort package.

**Comment by hnyman** [comment]
snort update has been superseded by #5702 , but is that libdaq update still relevant?  (related to snort?)

**Comment by MikePetullo** [comment]
Snort depends on libdaq. I suspect I will eventually put in a pull request to take over maintaining libdaq, but I have not gotten that far yet. For the time being, the snort pull request here is invalid, but the libdaq request could be split out.

**Comment by hnyman** [comment]
closed by  #5702 and 812ce161f3

**Pull request closed without merging**